packages/plugin-aime/src/providers/dnaProvider.ts

```diff
@@ -0,0 +1,55 @@
+import { Provider, IAgentRuntime, Memory, elizaLogger } from "@elizaos/core";
+
+const CHARACTER_ID = "b00918728e0a4e5f80f0d9233eca4040";
```

Comment:
This doesn't look right

Reply:
We have changed to fetch onchain DNA context by AIME_ONCHAIN_ADDRESS